app/helpers/blocks_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BlocksHelper
+  def current_block_check_box(block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `current_block_check_box(*)` if you want the method to accept any arguments but don't care about them.